src/extension/module.js

```diff
@@ -2715,6 +2749,7 @@
             await fetch(`https://${config.innerHost}${path}`, { cache: 'reload', mode: 'no-cors' });
           }
           fireEvent(this, 'updated', path);
+          sampleRUM('sidekick:preview');
```

Comment:
```suggestion
          sampleRUM('sidekick:preview', {
             source: '', // TODO: the current URL being edited or previewed (this can be the word or google docs URL)
             target: '' // TODO: the future `.live` URL of the page
          });
```